fundingmanager.go

```diff
@@ -2888,6 +2888,14 @@ func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
 		}
 	}
 
+	// If the caller specified to spend all funds, then we
+	// define the maximum allowed funding amount to be the
+	// current limit of the maximum funding amount.
+	var fundUpToMaxAmt btcutil.Amount
+	if msg.fundAll {
+		fundUpToMaxAmt = MaxFundingAmount
```

Comment:
should we return an error if the user attempts `fundAll` but have more funds than `MaxFundingAmount`?